Cargo.toml

```diff
@@ -153,6 +153,7 @@ object-store = { path = "src/object-store" }
 partition = { path = "src/partition" }
 promql = { path = "src/promql" }
 query = { path = "src/query" }
+raft-engine = { git = "https://github.com/tikv/raft-engine.git", rev = "2dcaf5beeea3d5de9ec9c7133a2451d00f508f52" }
```

Comment:
Just checked, raft-engine has released version 0.4.0. The revision "2dcaf5beeea3d5de9ec9c7133a2451d00f508f52" is included.